runtime/vm/MHInterpreter.cpp

```diff
@@ -64,6 +64,7 @@ static const char * names[] = {
 /* #ifdef J9VM_OPT_PANAMA */
 	"J9_METHOD_HANDLE_KIND_NATIVE",
 /* #endif J9VM_OPT_PANAMA */
+	"J9_METHOD_HANDLE_KIND_FILTER_ARGUMENTS_WITH_COMBINER",
```

Comment:
A placeholder is needed for `J9_METHOD_HANDLE_KIND_NATIVE` even if `J9VM_OPT_PANAMA` is not defined else this table will be too short.